packages/frontend/src/pages/Dashboard.tsx

```diff
@@ -268,18 +292,28 @@ const Dashboard = () => {
                     cols={{ lg: 12, md: 10, sm: 6 }}
                     layouts={layouts}
                 >
-                    {dashboardTiles.map((tile) => (
-                        <div key={tile.uuid}>
-                            <TrackSection name={SectionName.DASHBOARD_TILE}>
-                                <GridTile
-                                    isEditMode={isEditMode}
-                                    tile={tile}
-                                    onDelete={onDelete}
-                                    onEdit={onEdit}
-                                />
-                            </TrackSection>
-                        </div>
-                    ))}
+                    {dashboardTiles.map((tile) => {
+                        const tileUuid = (tile as DashboardChartTileProperties)
+                            ? // @ts-ignore
```

Comment:
why do we need this ignore here ?

Reply:
![Screenshot 2022-04-13 at 09 49 41](https://user-images.githubusercontent.com/31137824/163138164-d37324a7-2330-487f-b279-5f7031610267.png)

The actual ID i'm trying to get to is the `savedChartUuid` but this is only available when its a Cartesian, tile, if its a loom or markdown tile then i want to get the `tile.uuid` however TS wont stop complaining :'(

Reply:
Does that mean we can't duplicate BigNumber charts ? :/

Reply:
This is not relevant anymore, the issue i had heres was that i was trying to get the id of the chart/bignumber/table to dublicate, however this gets me the tile id not the actual chart id. so it was erroing as for instance `loom or markdown tiles` wont have the id i needed. 

However after a conversation with @PriPatel this is actually not on the scope neither something we need (duplicating the charts from within a dashboard) so this can now be ignored.